libass/ass_render.c

```diff
@@ -2600,7 +2601,7 @@ static void add_background(ASS_Renderer *render_priv, EventImages *event_images)
  * Process event, appending resulting ASS_Image's to images_root.
  */
 static bool
-ass_render_event(ASS_Renderer *render_priv, ASS_Event *event,
+ass_render_event(ASS_Track *track, ASS_Renderer *render_priv, ASS_Event *event,
```

Comment:
I think it's better to swap `track` and `render_priv` parameters to make it similar to `ass_render_frame()`.

Reply:
Adding `ASS_Track` to the arguments is superfluous. `ASS_Renderer *render_priv` already contains a reference to the associated track.